kagi/src/plans/why-pay-for-search.md

```diff
@@ -0,0 +1,29 @@
+> 
+> It is clear that a search engine which was taking money for showing cellular phone ads would have difficulty justifying the page that our system returned to its paying advertisers. For this type of reason and historical experience with other media [Bagdikian 83], we expect that advertising funded search engines will be inherently biased towards the advertisers and away from the needs of the consumers."
+> 
+> — [The Anatomy of a Large-Scale Hypertextual Web Search Engine](http://infolab.stanford.edu/pub/papers/google.pdf), Sergey Brin and Lawrence Page, 1998
+
+They were right in recognizing early on that ad-supported search engines will be biased towards the advertisers. They probably could not foresee how much damage to the web adopting the model they originally criticized would ultimately cause.
+
+## Ad-Supported Search Engines & How You Currently Pay for Search 
+According to Google Search Statistics the average person searches Google three or four times per day or 90-120 times per month. According to DuckDuckGo’s data, the monthly average is about 30 searches per person or one per day. These search results are substantially impacted by advertisers and affiliate links that pay Google to influence your results. The cost is not free, you are paying with your time and cognitive bandwidth required to parse the results to find what is actually relevant. Consider how much time you spend scrolling through initial results to pass over the results you know are meaningless. Also consider the personal information you are sharing and losing control of as you use the search services that present themselves as free.  
+
+## Kagi Strives to Lower Cost 
+All search engines have search costs, development costs, and administrative costs. Most search engines cover this by advertising, tracking, and selling your data. Currently Kagi spends about USD 1.25 cents per search however  search indexes are about to become several times more expensive and we now have the AI integration cost. It is our intention to keep the cost at USD 1.5 cents per search or below by rethinking our partnering strategy and investing more into our infrastructure. 
+
+## The Numbers Tell a Tale
+In 2022 Google generated USD $224.47 Billion dollars from advertisement revenue while processesing approximately 8 Billion searches per day. At 365 days per year this amounts to approximatley USD $0.07 revenue per search. If an average user searches 10 times per day, assuming a 30 day month this results in Google generating USD $21 revenue per user per month. However the value you receive is well below USD $21 because your results are influenced so that Google can create that type of revenue. Chosing to subscribe to Kagi means that while you are now paying for search you are getting a fair value for your money, you are getting more relevant results, are able to personalize your expeirence and take advantage of AI tools all while protecting your and your family's privacy and data. 
```

Comment:
```suggestion
In 2022 Google generated USD $224.47 Billion dollars from advertisement revenue while processing approximately 8 Billion searches per day. At 365 days per year this amounts to approximatley USD $0.07 revenue per search. If an average user searches 10 times per day, assuming a 30 day month this results in Google generating USD $21 revenue per user per month. However, the value you receive is well below USD $21 because your results are influenced so that Google can create that type of revenue. Chosing to subscribe to Kagi means that while you are now paying for search you are getting a fair value for your money, you are getting more relevant results, are able to personalize your experience and take advantage of AI tools all while protecting your and your family's privacy and data. 
```